metaci/testresults/robot_importer.py

```diff
@@ -276,3 +307,37 @@ def render_robot_test_xml(root, test):
 
     test_xml = ET.tostring(testroot, encoding="unicode")
     return re.sub(r"sid=.*<", "sid=MASKED<", test_xml)
+
+
+def export_robot_test_results(flowtest, test_results) -> None:
+    if not settings.METACI_RELEASE_WEBHOOK_URL or not flowtest:
+        return  # should we better error handle this for individual case message error handling?
+    logger.info(
+        f"Sending test results webhook for {flowtest.build_flow.build.get_external_url()} to {settings.METACI_RELEASE_WEBHOOK_URL}"
+    )
+    payload = {
+        "build": {
+            "name": flowtest.build_flow.build.plan.name,
+            "number": flowtest.id,
+            "url": "https://www.example.com/",  # flowtest.build_flow.build.get_external_url(),  # returns 'detail': [{'loc': ['body', 'build', 'url'], 'msg': 'URL host invalid, top level domain required', 'type': 'value_error.url.host'}] when sent to gus-bus,
+            "metadata": flowtest.build_flow.build.repo.metadata,
+        },
+        "tests": test_results,
+    }
+
+    token = jwt_for_webhook()
+    response = requests.post(
+        f"{settings.METACI_RELEASE_WEBHOOK_URL}/test-results/",
+        json=payload,
+        headers={"Authorization": f"Bearer {token}"},
+    )
+    result = response.json()
```

Comment:
Always do response.raise_for_status() before actually accessing the json -- to make sure we got an HTTP 200 response and not some HTTP error